<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>What is the text about ?</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLoadFilePosition">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadFile</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>192</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelWord">
       <property name="text">
        <string>Word</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtWord">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>        -------------------</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>192</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPosition">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPosition">
       <property name="placeholderText">
        <string>value position</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="txtGetFrecuency">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGetPositions">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>258</width>
      <height>215</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelTitulo">
       <property name="text">
        <string>Most Common Words of the Paper</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetWords"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>98</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btnGetFrecuency">
       <property name="text">
        <string>Get Frecuency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGetPosicion">
       <property name="text">
        <string>Get Posicions</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelMasFrecuentes">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>The n Most Repeated Words</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMasRepetidas">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetMasRepetidas">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetN">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumberOfWords">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Words</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPalabrasTotales">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="ActionOpenStopWords"/>
    <addaction name="ActionOpenPaper"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="ActionSave"/>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="ActionAboutAplication"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionOpenStopWords"/>
   <addaction name="ActionOpenPaper"/>
   <addaction name="ActionSave"/>
  </widget>
  <action name="ActionOpenStopWords">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/OpenImportant.jpeg</normaloff>:/rec/OpenImportant.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Stop Words</string>
   </property>
  </action>
  <action name="ActionOpenPaper">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/openFile.jpeg</normaloff>:/rec/openFile.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Paper</string>
   </property>
  </action>
  <action name="ActionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/save.jpeg</normaloff>:/rec/save.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="ActionAboutAplication">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/about.jpeg</normaloff>:/rec/about.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;About Aplication</string>
   </property>
  </action>
  <action name="ActionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/close.jpeg</normaloff>:/rec/close.jpeg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
